Tribler/Test/Core/test_session.py

```diff
@@ -89,6 +89,7 @@ def test_add_torrent_def_to_channel(self):
         """
         Test whether adding a torrent def to a channel works
         """
+        self.session.notifier.use_pool = False
```

Comment:
I will drop the commit from this PR and refactor the tests in another separate PR.